tests/testthat/test-prepare_matches_mapper.R

```diff
@@ -7,3 +7,31 @@ test_that("`completion` column has either `low`, `medium`, or `high` values", {
   out <- prepare_matches_mapper(small_matches_mapper, ecoinvent_activities)
   expect_equal(unique(out$completion), c("low", "high", "medium"))
 })
+
+test_that("`prepare_matches_mapper` doesn't have any empty string", {
+  tibble_names <- function(x, nms) {
+    m <- matrix(x, ncol = length(nms), dimnames = list(NULL, nms))
+    tibble::as_tibble(m)
+  }
+
+  mm <- tibble_names("a", names(matches_mapper))
+  mm$activity_uuid_product_uuid <- NA_character_
+  ea <- tibble_names("a", names(ecoinvent_activities))
+
+  out <- prepare_matches_mapper(mm, ea)
+  expect_equal(out$activity_uuid_product_uuid, NA_character_)
+})
+
+test_that("`multi_match` column outputs `FALSE` for null input value", {
+  tibble_names <- function(x, nms) {
+    m <- matrix(x, ncol = length(nms), dimnames = list(NULL, nms))
+    tibble::as_tibble(m)
+  }
+
+  mm <- tibble_names("a", names(matches_mapper))
+  mm$multi_match <- NA_character_
+  ea <- tibble_names("a", names(ecoinvent_activities))
+
+  out <- prepare_matches_mapper(mm, ea)
+  expect_equal(unique(out$multi_match), c("FALSE"))
```

Comment:
So we expect "FALSE" not `FALSE`?

``` r
devtools::load_all()
#> ℹ Loading tiltIndicatorAfter

tibble_names <- function(x, nms) {
  m <- matrix(x, ncol = length(nms), dimnames = list(NULL, nms))
  tibble::as_tibble(m)
}

mm <- tibble_names("a", names(matches_mapper))
mm$multi_match <- NA_character_
ea <- tibble_names("a", names(ecoinvent_activities))

out <- prepare_matches_mapper(mm, ea)

testthat::expect_false(out$multi_match)
#> Error: out$multi_match is not FALSE
#> 
#> `actual` is a character vector ('FALSE')
#> `expected` is a logical vector (FALSE)
```

<sup>Created on 2023-10-12 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

Reply:
We expect "FALSE" and not logical FALSE

Reply:
@maurolepore I have changed it to `FALSE` (logical)